src/open_samus_returns_rando/pickups/pickup.py

```diff
@@ -12,14 +12,9 @@
 from open_samus_returns_rando.logger import LOG
 from open_samus_returns_rando.lua_editor import LuaEditor, ScriptClass
 from open_samus_returns_rando.patcher_editor import PatcherEditor, path_for_level
+from open_samus_returns_rando.pickups.map_icons import get_map_icon_data
 from open_samus_returns_rando.pickups.model_data import get_data
 
-RESERVE_TANK_ITEMS = {
-    "ITEM_RESERVE_TANK_LIFE": "powerup_energyreservetank",
-    "ITEM_RESERVE_TANK_SPECIAL_ENERGY": "powerup_aeionreservetank",
-    "ITEM_RESERVE_TANK_MISSILE": "powerup_missilereservetank",
-}
-
```

Comment:
This breaks old patcher files, but I have no idea why we had a mapping for the item id to a model instead of using the correct model in rdv in the first place

Reply:
Oh wait, was this actually every released that was?

Reply:
They were just assigned the tank model, so I had to use the id to determine if it should have the fx or not. Not the best solution. It's been in releases yes. Don't know how much we care about breaking older patcher for this. I personally don't care but eh.

Reply:
:(
In retrospect this should have just been a new model name like it is now.
Iirc so far we always bumped the major version if we had breaking changes for old input files. Then I revert that part.